app_handler.go

```diff
@@ -37,11 +37,10 @@ func (app *App) handler() func(*fasthttp.RequestCtx) {
 		}
 		app.middlewaresMu.RUnlock()
 		if len(app.domains) > 0 {
-			d := string(ctx.URI().Host())
 			app.domainListLock.RLock()
-			if app.domains[d] != nil {
+			if app.domains[string(ctx.URI().Host())] != nil {
```

Comment:
please revert this change. the `d` variable was used to avoid unnecessary allocations. you can rename it to `domain` if you'd like to, but I think we should keep the var